presto/README.md

```diff
@@ -46,13 +46,13 @@ logs_enabled: true
 ```
 logs:
   - type: file
-    path: /data/var/log/*.log
+    path: /var/log/presto/*.log
```

Comment:
I understand the /presto/ addition, but why the /data/ removal?

Reply:
@hithwen as you can see on this [documentation](https://github.com/prestodb/presto/blob/master/presto-server-rpm/README.md#rpm-package-build-and-usage), the actual default path for the file is indeed `/var/log/presto` which is why it is changed.